### ISSUE TITLE
How do I get version 0.75?

### ISSUE THREAD

**Issue by tcompart** [drolbr/Overpass-API#457]
I used branch new_features and hoped to get at least API version 0.75, however, running the dispatcher starts then the 0.6 api version, which would be served via my local nginx. 
The db file is as osm3s_v0.7.54_osm_base written. How do I get the 0.75 api connected with the database?

any help would be appreciated. I have not found any information how to switch in the README (which just links to the wiki).

**Comment by mmd-osm** [comment]
0.7.55 and the new features branch are still in development and not officially released. I‘d suggest to wait some more time...

**Comment by tcompart** [comment]
so it is no possible to access the API on the 0.7 version during the development from any commit or branch?

**Comment by mmd-osm** [comment]
No, api_mmd runs on a fork which is also not released at this time: https://github.com/mmd-osm/Overpass-API/tree/test758_lz4hash 

Travis ci script has all steps to compile this fork.

**Comment by mmd-osm** [comment]
We've noticed that you sent about 67'000 requests to our development server today without asking for prior permission. Please refrain from similar activities in the future without explicit consent.

**Comment by tcompart** [comment]
Hi,
thank you for the link to the fork of this repo. I will try to build it myself. And sorry about the huge number of requests. That was not our intention to produce that many requests. Hopefully I can create the mmd release myself and can give something in return! Thank you! For now we should have stopped the number of requests.
